allennlp/common/testing/distributed_test.py

```diff
@@ -69,5 +69,5 @@ def run_distributed_test(
         init_process,
         args=(device_ids, world_size, func, args, kwargs),
         nprocs=nprocs,
-        start_method="fork",
+        start_method="spawn",
```

Comment:
Need to use spawn as the start method in order to run these tests on GPU for some reason.

Reply:
Fwiw, using the `spawn` method was causing some pickle-related error, when I was working on this. https://github.com/pytorch/pytorch/issues/22585

Reply:
Hmm darn! Ok, I'll take another look

Reply:
Okay this is ready for another look @AkshitaB